infrastructure/tooling/src/generate/generators/docker-compose.js

```diff
@@ -340,6 +340,9 @@ exports.tasks.push({
         ui: serviceDefinition('ui', {
           command: 'ui/web',
           _useEnvFile: true,
+          volumes: [
+            './docker/nginx-ui.conf:/app/ui/web-ui-nginx-site.conf',
```

Comment:
this is the least intrusive change imho.

I also tried `sed -i \'/Content-Security-Policy/d\' /app/ui/web-ui-nginx-site.conf;` but then realised if someone would like to use `docker compose` to run TC in production (for some strange reason), at least it wouldn't strip all of the security headers. And since it is an extra config file, it can be overwritten or volume mounts could be removed